index.html

```diff
@@ -9,6 +9,29 @@
         <link rel="stylesheet" href="main.css">
     </head>
     <body>
-        
+        <nav>
+            <p>Welcome</p>
```

Comment:
- [ ] Consider wrapping this logo in an `a` tag so that when user clicks on your logo, it takes them to your home page 😄. See suggestion below:

```suggestion
         <a href="#">
            <p>Welcome</p>
          </a>
```